playbooks/scale-up.yaml

```diff
@@ -0,0 +1,43 @@
+---
+# Set new number of nodes
+- hosts: masters
+  tasks:
+  - name: Get number of nodes
+    shell: oc get nodes | grep app-node -c
+    register: oc_old_result
+  - name: Set new number of nodes
+    set_fact:
+      # Number of nodes in the new configuration (incremented by 1 by default)
+      nodes: "{{ oc_old_result.stdout | int + 1 }}"
+  - name: Check that new value is higher
```

Comment:
I'm confused now. If the number of nodes may change runtime, it makes no sense to verify that the `oc get nodes` value matches the existing `openstack_num_node`? Otherwise, if it never changes run-time, it doesn't make sense to run `oc get nodes`, as it will be  always equal to the `openstack_num_node`...